samples/Koka/LICENSE

```diff
@@ -0,0 +1,210 @@
+Copyright 2012-2023, Microsoft Research, Daan Leijen.
```

Comment:
We don't want this file. All samples are used to train the classifier and this is clearly not written in the language you are adding 😉

Reply:
Thanks. How would you suggest including the entire license then?

I want to ensure we comply with copyright law :sweat_smile:

Reply:
We don't need it. Were only need the license stated in the PR template.